lib/pymedphys/tests/icom/test_icom_cli.py

```diff
@@ -0,0 +1,64 @@
+# Copyright (C) 2021 Radiotherapy AI Pty Ltd
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import lzma
+import multiprocessing
+import os
+import pathlib
+import socket
+import subprocess
+import sys
+import tempfile
+
+from pymedphys._data import download
+from pymedphys._root import LIBRARY_ROOT
+
+
+def test_icom_cli():
+    icom_server_process = multiprocessing.Process(target=_mock_icom_server)
+    icom_server_process.start()
+
+    env = os.environ.copy()
+    if getattr(sys, "frozen", False):
+        env["PATH"] = str(LIBRARY_ROOT.parents[1]) + os.pathsep + env["PATH"]
```

Comment:
It would be helpful to explain what this parents[1] is pointing to and why.  I get the gist of it, but...

Reply:
Yup, I completely agree. That could be much clearer. Made it explicit in:

https://github.com/pymedphys/pymedphys/pull/1557/commits/ae3309c44cadd52d0e20a6422c2f9a09ba29f161

Thanks @sjswerdloff :slightly_smiling_face: